config/env/experimental.env

```
@@ -24,3 +24,4 @@ DB_IAM=true
 DB_REGION=us-west-2
 DB_IAM_ROLE=arn:aws:iam::923914045601:role/ecs-rds-role-app-experimental
 DEBUG_PPROF=false
+AWS_CF_DOMAIN=assets.experiemntal.move.mil
```

```suggestion
AWS_CF_DOMAIN=assets.experimental.move.mil
```

I'm curious how this worked if this was broken. Can you try again with another deploy?

---

ahhh I have a hunch: https://github.com/transcom/mymove/pull/3096/files#diff-a48e01cd35e48856f1bd32297b21388fR170-R171 are these values overwriting the env value?